redfish/provider/resource_redfish_storage_volume.go

```diff
@@ -434,13 +481,14 @@ func createRedfishStorageVolume(ctx context.Context, service *gofish.Service, d
 	}
 
 	newVolume := map[string]interface{}{
-		"VolumeType":         volumeType,
 		"DisplayName":        volumeName,
 		"Name":               volumeName,
 		"ReadCachePolicy":    readCachePolicy,
 		"WriteCachePolicy":   writeCachePolicy,
 		"CapacityBytes":      capacityBytes,
 		"OptimumIOSizeBytes": optimumIOSizeBytes,
+		"RAIDType":           raidType,
+		"Encrypted":          encrypted,
```

Comment:
Looks like this change has effectively removed the functionality of the "volume_type" field instead of just marking it as deprecated. That means that this is a breaking change. The backward-compatible way to do this would be to check which field is actually "known" and construct the payload.

Reply:
Hey @rounak-adhikary , No it is deprecated.
We just translated the `volume_type` into `raid_type` above. 

So if the user still has `volume_type` in there configs it will still work in 1.2.0 it will just show as a `warning` with the deprecated method asking the user to update. See line `440` and the new `volumeTypeMap` for more info.

Reply:
I did make `volume_type` optional and the new `raid_type` is optional at least during the transition?
I am a little confused I did thing thing discussed above? 

Is there something specifically, You wanted to mention?

Reply:
Oh sorry. I see what u mean by translating volume type into raid type. Missed that before.